### PR TITLE
Fix primitive coercion on platforms that don't support Symbol.toPrimitive.

### LINKED ISSUE #74: Symbol.toPrimitive doesn't exist in IE11
https://github.com/GoogleChromeLabs/jsbi/blob/c80c3ae61ca16fd37be76671fcf33df494369b34/lib/jsbi.ts#L1764-L1769

My understanding is that this code won't work in IE11 as neither Symbol or Symbol.toPrimitive are defined there.

Is using toPrimitive necessary, or could it be removed? If it is needed, then I would suggest wrapping it in some typeof checks:

```
if (typeof Symbol !== "undefined" && typeof Symbol.toPrimitive === "symbol" ) {
   ... existing code here ...
}
```

I can send a PR for the above, but wanted to check and see whether this code is even needed before continuing down this path.

### PR DESCRIPTION
**Pull request by 12wrigja**
Fixes #74.

### CODE REVIEW

File: tests/tests.mjs [hunk 1]
```diff
@@ -46,6 +46,22 @@ import JSBI from '../jsbi';
...
   console.assert(JSBI.EQ(JSBI.BigInt(18014398509481988), 18014398509481988));
+
+  // Emulate an environment that doesn't have Symbol (e.g. IE11)
+  // and make sure we can still coerce to primitive values.
+  // See #74.
+  let globalSymbol = globalThis.Symbol;
+  try {
+    globalThis.Symbol = undefined;
+    console.assert(JSBI.EQ(JSBI.BigInt(0x7FFFFFFF), {
+      valueOf: () => 0x7FFFFFFF
```
Comment (mathiasbynens) [inline tests/tests.mjs:57]:
```suggestion
      valueOf: () => 0x7FFFFFFF,
```

File: tests/tests.mjs [hunk 2]
```diff
@@ -46,6 +46,22 @@ import JSBI from '../jsbi';
   console.assert(JSBI.EQ(JSBI.BigInt(18014398509481980), 18014398509481980));
   console.assert(JSBI.EQ(JSBI.BigInt(18014398509481982), 18014398509481982));
   console.assert(JSBI.EQ(JSBI.BigInt(18014398509481988), 18014398509481988));
+
+  // Emulate an environment that doesn't have Symbol (e.g. IE11)
+  // and make sure we can still coerce to primitive values.
+  // See #74.
+  let globalSymbol = globalThis.Symbol;
```
Comment (mathiasbynens) [inline tests/tests.mjs:53]:
```suggestion
  const globalSymbol = globalThis.Symbol;
```

**Review by mathiasbynens — APPROVED** [review]

**Review by jakobkummerow — APPROVED** [review]
LGTM, thanks!

File: tests/tests.mjs (hunk 2 above)
Comment (12wrigja) [reply tests/tests.mjs]: Done.

File: tests/tests.mjs (hunk 1 above)
Comment (12wrigja) [reply tests/tests.mjs]: Done.

**Pull request merged**